src/uu/cp/src/cp.rs

```diff
@@ -585,7 +585,24 @@ impl Options {
             || matches.is_present(OPT_RECURSIVE_ALIAS)
             || matches.is_present(OPT_ARCHIVE);
 
-        let backup = matches.is_present(OPT_BACKUP) || (matches.occurrences_of(OPT_SUFFIX) > 0);
+        let backup_mode = backup_control::determine_backup_mode(
+            matches.is_present(OPT_BACKUP),
+            matches.value_of(OPT_BACKUP),
+        );
+        let backup_suffix = backup_control::determine_backup_suffix(matches.value_of(OPT_SUFFIX));
+
+        let overwrite = OverwriteMode::from_matches(matches);
+
+        if overwrite == OverwriteMode::NoClobber && backup_mode != BackupMode::NoBackup {
+            show_error!(
```

Comment:
Ah I missed this on the previous review. You can use the `show_usage_error` here.

Reply:
Got it, I moved this check to `uumain` so I could use `show_usage_error` and `return 1` instead of returning an error from `Options::from_matches`, which would result in weird output from the usage error and the returned error.

Reply:
I also added a test (`test_cp_backup_no_clobber_conflicting_options`) to check for the correct stderr from the `cp --backup --no-clobber` conflicting error.